ionic_user_interface/src/providers.ts

```diff
@@ -5,7 +5,7 @@ import jwt_decode from 'jwt-decode';
 import { toastController } from '@ionic/vue';
 
 const isLoggedIn = ref(false);
-const userEmail = ref('');
+const username = ref('');
```

Comment:
No need for this anymore, I was using userEmail to display their email in Confirm.vue. Best if we can get the userEmail from the server when "UserNotConfirmed" is returned

Reply:
`username` is needed now as a param when confirming. I will try to return email

Reply:
Okie done